packages/explat/src/index.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { createExPlatClient } from '@automattic/explat-client';
+import createExPlatClientReactHelpers from '@automattic/explat-client-react-helpers';
+
+/**
+ * Internal dependencies
+ */
+import { isDevelopmentMode } from './utils';
+import { logError } from './error';
+import { fetchExperimentAssignment } from './assignment';
+import { getAnonId, initializeAnonId } from './anon';
+
+declare global {
+	interface Window {
+		wcTracks: {
+			isEnabled: boolean;
+		};
+	}
+}
+
+export const initializeExPlat = (): void => {
+	if ( window.wcTracks?.isEnabled ) {
```

Comment:
Are we sure `isEnabled` is set within the `.enabled()` callback?

Reply:
Yeah, thats a WC core function that sets `isEnabled` to `true`, loads `w.js` and executes the callback once `w.js` loads

https://github.com/woocommerce/woocommerce/blob/9398c894c943524928d5b181eaeba9b44eb7a8ad/includes/tracks/class-wc-site-tracking.php#L107-L108